src/main/java/module/series/MatchFixtures.java

```diff
@@ -297,46 +297,57 @@ private List<Paarung> getMatchesByTeamIds(List<Integer> ids) {
      * @param currentTeams  List of the current teams
      * @param t             The examined team
      */
-    private void FindReplacementOfTeam(ArrayList<List<Integer>> currentTeams, Integer t) {
-        for ( var ids : currentTeams) {
-            if ( ids.contains(t)) {return;} // not replaced
+    private void findReplacementOfTeam(ArrayList<List<Integer>> currentTeams, Integer t) {
+        for (var ids : currentTeams) {
+            if (ids.contains(t)) {
+                return;
+            } // not replaced
         }
 
         // Find replacement of team
         List<Integer> replaces = new ArrayList<>();
         replaces.add(t);
 
-        while ( !replaces.isEmpty() ) {
+        while (!replaces.isEmpty()) {
             var replaceTeam = replaces.get(0);
             for (var i = 0; i < 7; i++) {
                 int finalI = i;
                 var match = m_vEintraege.stream().filter(p -> p.getSpieltag() == 1 + finalI && (p.getHeimId() == replaceTeam || p.getGastId() == replaceTeam)).findAny();
-                if ( match.isPresent()) {
+                if (match.isPresent()) {
                     // Find reverse match
-                    if ( match.get().getHeimId() == replaceTeam) {
+                    if (match.get().getHeimId() == replaceTeam) {
                         var opponentAtRound = match.get().getGastId();
                         var reverseMatch = m_vEintraege.stream().filter(p -> p.getSpieltag() == 14 - finalI && (p.getHeimId() == opponentAtRound)).findAny();
-                        if ( reverseMatch.isPresent()) {
+                        if (reverseMatch.isEmpty()) {
```

Comment:
Ah got it, does this mean that there were more than 1 team replaced in that league?  Wow, I don't think I had appreciated the complexity of this feature...

Reply:
Yes exactly - 2 teams of that league were replaced